protocol/communities/community.go

```diff
@@ -1255,6 +1256,22 @@ func (o *Community) PrivateKey() *ecdsa.PrivateKey {
 	return o.config.PrivateKey
 }
 
+func (o *Community) setPrivateKey(pk *ecdsa.PrivateKey) {
+	if pk != nil {
+		o.config.PrivateKey = pk
+	}
+}
+
+func (o *Community) ControlNode() *ecdsa.PublicKey {
+	return o.config.ControlNode
+}
+
+func (o *Community) setControlNode(pubKey *ecdsa.PublicKey) {
+	if pubKey != nil {
+		o.config.ControlNode = pubKey
+	}
+}
```

Comment:
will this be changed? In my understanding, control node can be only one device, but in this case all devices will be treated as a control node because they all will have the same pubkey

Reply:
I believe that's fine @osmaczko is working on some changes that would allow this. The key is set to the chat key of the user, so it's already shared across devices.